site/snippets/templates/buy/good-cause.php

```diff
@@ -29,8 +29,8 @@
 									'width' => 300
 								],
 								'srcset' => [
-									'1x' => 400,
-									'2x' => 800,
```

Comment:
`Question:` You wrote `1x`/`2x` aren't needed/used together with `sizes`. But given that no sizes are passed here, wouldn't it still help for all browsers not supporting `sizes="auto`?

Reply:
Yes, if the browser doesn't support `sizes="auto"`, the widths won't be useful. However I feel like the `2x` size at the moment is also not useful as none of our images render at a fixed size. A 1x device may still need a larger version while a 2x device could end up needing the smaller one. Best way would be to add `sizes` manually everywhere, but I don't think it's worth it for the transition. The current changes are already an improvement over the status quo.